### PR TITLE
Separating slow tests from faster running tests

### PR DESCRIPTION
**Pull request by RabeaGleissner**
@felipesere @jsuchy

I moved the two slower running tests to a separate file and am using the Maven Surefire plugin to exclude that file from the default test run. To run all tests, the command `mvn integration-test` needs to be used.

### CODE REVIEW

File: src/test/java/de/rabea/player/ComputerPlayerSpeedTest.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        UnbeatableComputerPlayer computer = new UnbeatableComputerPlayer(X);
+        Board board;
+        long startTime = System.nanoTime();
+        for (int i = 0; i <= 10; i++) {
+            board = new Board(3);
+            computer.getPosition(board);
+        }
+        long endTime = System.nanoTime();
+        long duration = (endTime - startTime)/1000000;
+        System.out.println("duration in milliseconds= " + duration);
```
Comment (jsuchy) [inline src/test/java/de/rabea/player/ComputerPlayerSpeedTest.java]: Not sure about leaving these println statements in these tests. At the very least, when you run all the slow tests, there's no context about what each of the durations are for.

Comment (RabeaGleissner) [inline src/test/java/de/rabea/player/ComputerPlayerSpeedTest.java]: @jsuchy took them out :+1:

**Comment by felipesere** [comment]
I would take this one step further:
Create a function that takes a board an measures how long it takes, then use it with `new Board(3)` and `new Board(4)` respectively. The duplication between those two tests is not needed.

**Comment by RabeaGleissner** [comment]
@felipesere Thanks for the feedback. I've removed the duplication. I also changed the test for 3x3 to only run the `computer.getPosition()` method once.

**Comment by felipesere** [comment]
Its looking good.
@jsuchy question about taste: not having an `assert` call looked weird to me. But now that I think of it, the assert will never 'fail' as the the `@Timeout(...)` will trigger first. 
Would you leave the assertion in anyway?

**Comment by jsuchy** [comment]
I would kill assertions (similar to a test that ensures an exception is thrown, the "assertion" is in the annotation).

**Pull request merged**